app/gui/src/dashboard/providers/BackendProvider.tsx

```diff
@@ -102,23 +102,36 @@ export function useLocalBackend() {
 
 /**
  * Get the corresponding backend for the given property.
- * @throws {Error} when neither the Remote Backend nor the Local Backend are supported.
+ * @throws {Error} when the Local Backend is not supported.
```

Comment:
still not quite accurate, if it doesn't ask for a local category it doesn't throw

Reply:
I think I do not quiet understand, couuld you elaborate? We throw an errob, but it's within the pickBackend. I think thats fair to add the throw line here...

Reply:
yeah it *is*, but it doesn't unconditionally throw when `LocalBackend` is `null`, only when you're trying to access a local category when it is `null`, right?

Reply:
Thanks, fixed